org.mozilla.vpn.yml

```diff
@@ -0,0 +1,144 @@
+app-id: org.mozilla.vpn
+runtime: org.kde.Platform
+runtime-version: '6.9'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: mozillavpn
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --system-talk-name=org.freedesktop.NetworkManager
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  env:
+    - CARGO_HOME=/run/build/rust-flatpak/cargo
```

Comment:
```suggestion
```

the syntax is `CARGO_HOME: /run/build/$module_cargo/cargo` but looks like this is from a template, there is no module called rust-flatpak.

it is unused.